backend-shared/plannershared.cpp

```diff
@@ -199,7 +199,9 @@ void plannerShared::set_min_switch_duration(int value)
 
 double plannerShared::bottomsac()
 {
-	return qPrefDivePlanner::bottomsac() / 1000.0;
+	return (qPrefUnits::volume() == units::LITER) ?
+				qPrefDivePlanner::bottomsac() / 1000.0 :
+				ml_to_cuft(qPrefDivePlanner::bottomsac());
```

Comment:
I find this an odd choice. We try to do everything in our native units and then do the conversions close to the UI layer. We often use helper functions to do this.
Of course that isn't consistent and the logic has been broken many times all over the code. And the return value here is `double` which isn't implying any unit in the first place. But still, this to me seems unintuitive.

Reply:
Please remember most of this is copied 1-1 from the desktop-widgets/diveplanner. And this is actually very close to the UI layer, unless you want to do it twice once for desktop-widgets and once for mobile-widgets. 
Actually in this particular case it might make sense to move it, because while the desktop UI spin box handles double, QML does not so that value need an extra conversion, just to complicate matters.